_includes/releases/v22.1/v22.1.0-beta.1.md

```diff
@@ -6,90 +6,75 @@ Release Date: April 4, 2022
 
 <h3 id="v22-1-0-beta-1-enterprise-edition-changes">Enterprise edition changes</h3>
 
-- Tenant GC job will now wait for protected timestamp records that target the tenant and have a protect time less than the tenant's drop time. Release justification: high-priority changes in existing functionality [#78389][#78389]
-- Allow users to provide an end-time for changefeeds through the 'end_time' option. When this option is provided, the changefeed will run until it has reached the end-time, and then the changefeed job will end with a successful status code. Furthermore, we now provide an 'initial_scan_only' option. When this option is set, the changefeed job will end with a successful status code immediately after the initial scan completed.  Release Justification: This feature is low danger, yet highly desired from users. [#78381][#78381]
-- Do not block schema changes when executing core-style changefeeds.  Release Justification: low danger stability fix. [#78360][#78360]
-- Remove expensive, unnecessary, and never used `schedules.round.schedules-ready-to-run` and `schedules.round.num-jobs-running` metrics from job schedulers.  Release Justification: Stability fix for scheduled job system. [#78565][#78565]
-- Changefeeds can now protect targets from GC on user tenants [#78536][#78536]
+- Tenant GC job will now wait for protected timestamp records that target the tenant and have a protect time less than the tenant's drop time. [#78389][#78389]
+- Allow users to provide an end time for [changefeeds](../v22.1/change-data-capture-overview.html) through the 'end_time' option. When this option is provided, the changefeed will run until it has reached the end timestamp the user specified, and then the changefeed job will end with a successful status code. Furthermore, we now provide an 'initial_scan_only' option. When this option is set, the changefeed job will run until the initial scan has completed, and then end with a successful status code. [#78381][#78381]
+- Do not block schema changes when executing core-style changefeeds. [#78360][#78360]
 
 <h3 id="v22-1-0-beta-1-sql-language-changes">SQL language changes</h3>
 
-- Add support for `ALTER DATABASE ... ALTER SUPER REGION`.  This command allows the user to change the regions of an existing super region.  
-	
-	For example, after successful execution of the following, super region `test1` will consist of three regions, `ca-central-1`, `us-west-1`, and `us-east-1`.
+- Added support for `ALTER DATABASE ... ALTER SUPER REGION`.  This command allows the user to change the regions of an existing super region. For example, after successful execution of the following, super region `test1` will consist of three regions, `ca-central-1`, `us-west-1`, and `us-east-1`.
 
-	`ALTER DATABASE db3 ALTER SUPER REGION "test1" VALUES "ca-central-1", "us-west-1", "us-east-1";`
+{% include_cached copy-clipboard.html %}
+```shell
```

Comment:
This needs to be sql for the syntax highlighting and we use ~~~ for the code block

Reply:
The code block and the sentence after need to be indented to match the bullet